src/rules/parameter-properties.ts

```diff
@@ -7,6 +7,17 @@ export const rule = createRule({
 				context.report({
 					messageId: "parameterProperty",
 					node,
+					suggest: [
+						{
+							fix(fixer) {
```

Comment:
[Bug] As with the docs, this should also add the equivalent property to the class. Otherwise the suggestion is removing a property.